### PR TITLE
Split pull request templates into 3 files

### PR DESCRIPTION
**Pull request by mohsen1**
This will make it easier to make a new PR without too much clutter

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/change_type_package.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+    Please fill in this template.
+-->
+
+## Checklist
+
+-   [ ] Use a meaningful title for the pull request. Include the name of the package modified.
```
Comment (DanielRosenwasser) [inline .github/PULL_REQUEST_TEMPLATE/change_type_package.md]: Could you fix the extra spacing?

**Review by DanielRosenwasser — COMMENTED** [review]
Great idea.

Just as a heads up, your commits don't seem to be associated with your GitHub account. While this isn't technically a problem, you might care if you want more appropriate attribution. You can either make sure your GitHub account is associated with the email address you're using for your commits, or rebase and amend your commits to fix the author name and email.

If you don't care about any of this stuff, I can always just pull in the changes.

**Comment by mohsen1** [comment]
According to [Github documentations](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) for pull request this will not work like issues where user will see 3 buttons for creating a pull request. The way they can use it is to pass query argument in the URL which I think is not ideal for this project.

**Comment by DanielRosenwasser** [comment]
:(

**Comment by mohsen1** [comment]
yup :(

**Pull request closed without merging**